.travis.yml

```diff
@@ -1,6 +1,9 @@
 sudo: required
 dist: trusty
 
+notifications:
+  email: false
```

Comment:
The more important thing here is running TSLint during the build so I can take this commit out